pkgs/development/python2-modules/scandir/default.nix

```diff
@@ -1,14 +1,17 @@
-{ lib, python, buildPythonPackage, fetchPypi }:
+{ lib, stdenv, python, buildPythonPackage, fetchPypi }:
 
 buildPythonPackage rec {
   pname = "scandir";
   version = "1.10.0";
 
   src = fetchPypi {
     inherit pname version;
-    sha256 ="1bkqwmf056pkchf05ywbnf659wqlp6lljcdb0y88wr9f0vv32ijd";
+    sha256 = "1bkqwmf056pkchf05ywbnf659wqlp6lljcdb0y88wr9f0vv32ijd";
   };
 
+  # https://github.com/NixOS/nixpkgs/issues/156807
+  doCheck = stdenv.hostPlatform.system != "aarch64-darwin";
+
```

Comment:
Since only some of the tests fail, it seems like it'd be better to just disable the ones that do? (There's a list of them in the test output in https://github.com/NixOS/nixpkgs/issues/156807).

Reply:
I think it will be much simpler to disable it, since `checkPhase` has been overridden, so `disabledTests` has no effect.

Reply:
Switched to `python -m unittest`, so that at least `test_walk.py` would be tested on aarch64-darwin.

Reply:
Patched.